src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -42,6 +42,26 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('terminal.integrated.fontFamily', "The font family used by the terminal (CSS font-family format)."),
 			'type': 'string',
 			'default': 'Menlo, Monaco, Consolas, "Droid Sans Mono", "Courier New", monospace, "Droid Sans Fallback"'
+		},
+		'terminal.integrated.fontSize': {
+			'description': nls.localize('terminal.integrated.fontSize', "The font size used by the terminal (CSS font-size format)."),
+			'type': 'string',
+			'default': ''
+		},
+		'terminal.integrated.lineHeight': {
+			'description': nls.localize('terminal.integrated.lineHeight', "The line height used by the terminal (CSS line height format)."),
+			'type': 'string',
+			'default': ''
+		},
+		'terminal.integrated.charWidth': {
```

Comment:
I also thought about that, but I have no clean solution in mind...
Perhaps measuring a character like X or M with canvas api... smth. like https://msdn.microsoft.com/en-us/library/ff975421(v=vs.85).aspx